### PR TITLE
fix Helpers.fetchUrl.phpt, http://example.com/404 returns 500 instead…

### PR DESCRIPTION
**Pull request by zdenekgebauer**
http://example.com/404 returns 500 instead 404

### CODE REVIEW

**Comment by dg** [comment]
What is use-case for `alwaysUpload`?

File: src/Deployment/CliRunner.php [hunk 1]
```diff
@@ -140,6 +140,9 @@ private function createDeployer($config)
 		$deployment->runAfterUpload = self::toArray($config['afterupload'], TRUE);
 		$deployment->runAfter = self::toArray($config['after'], TRUE);
 		$deployment->testMode = !empty($config['test']) || $this->mode === 'test';
+		$deployment->filePerms =  empty($config['filePerms']) ? '' : $config['fileperms'];
```
Comment (zdenekgebauer) [inline src/Deployment/CliRunner.php]: Yes, it is typo. Maybe config parameters should be the same in *.php and .ini files including capitalization?  (i.e.: afterupload vs afterUpload)

**Comment by zdenekgebauer** [comment]
I have a deploy process, where I need to run some scripts which delete few previously uploaded files. These files remains in .deployment file, and next deploy will not upload them.  Upload these files on each deploy with "alwaysUpload" is simpler solution than changing deploy process. 
I am not sure if this feature can be useful for more people. If you don't think so, ignore it.

**Comment by dg** [comment]
I added `upload:` job, it can solve your use case https://github.com/dg/ftp-deployment/commit/5128e8e5c489cbaa766ddb3d99f08c324cb3f893

**Pull request closed without merging**